patches/0012-Ensure-servicebus-queue-topic-have-namespaces-and-RG.patch

```diff
@@ -7,19 +7,19 @@ Subject: [PATCH] Ensure servicebus queue & topic have namespaces and RG names
 These are computed attributes for use in the servicebus mixins in Pulumi.
 
 diff --git a/internal/services/servicebus/servicebus_queue_resource.go b/internal/services/servicebus/servicebus_queue_resource.go
-index 579c5712ce..cc468319b5 100644
+index b4df3f33be..fc4838204d 100644
 --- a/internal/services/servicebus/servicebus_queue_resource.go
 +++ b/internal/services/servicebus/servicebus_queue_resource.go
-@@ -47,6 +47,8 @@ func resourceServiceBusQueue() *pluginsdk.Resource {
+@@ -46,6 +46,8 @@ func resourceServiceBusQueue() *pluginsdk.Resource {
```

Comment:
Just line # churn?

Reply:
Yup, think so - the there's no diff of the actual line changes